internal/expand/handler_test.go

```diff
@@ -35,6 +35,16 @@ func TestRESTHandler(t *testing.T) {
 	ts := httptest.NewServer(r)
 	defer ts.Close()
 
+	t.Run("case=returns required parameter \"max-depth\" is missing", func(t *testing.T) {
+		resp, err := ts.Client().Get(ts.URL + expand.RouteBase)
+		require.NoError(t, err)
+
+		assert.Equal(t, http.StatusBadRequest, resp.StatusCode)
+		body, err := io.ReadAll(resp.Body)
+		require.NoError(t, err)
+		assert.Contains(t, string(body), "required parameter \"max-depth\"")
```

Comment:
The tests are failing here now because the body is a JSON object, so the string escapes the quotes.